src/sinks/mod.rs

```diff
@@ -133,200 +133,249 @@ pub enum HealthcheckError {
 #[serde(tag = "type", rename_all = "snake_case")]
 #[enum_dispatch(SinkConfig)]
 pub enum Sinks {
-    /// AMQP.
+    /// Send events to AMQP 0.9.1 compatible brokers like RabbitMQ.
     #[cfg(feature = "sinks-amqp")]
+    #[configurable(metadata(docs::label = "AMQP"))]
     Amqp(amqp::AmqpSinkConfig),
 
-    /// Apex Logs.
+    /// Deliver log events to Apex.
```

Comment:
**note (non-blocking)**: Apex shut down last year and we'll probably remove it from Vector soon.

Reply:
Gotcha, thanks. We'll cross that bridge soon 🌉